websites/A/Animelizm/dist/metadata.json

```diff
@@ -0,0 +1,20 @@
+{
+    "author": {
```

Comment:
add `"$schema": "https://schemas.premid.app/metadata/1.0",` to the metadata

Reply:
Done